src/app/zeko/zkapps_rollup.ml

```diff
@@ -408,12 +408,14 @@ module Outer = struct
       { ledger_hash : Ledger_hash.t  (** The ledger hash of the rollup *)
```

Comment:
This part should be in a separate PR.

Reply:
The PR than won't be functioning, the sequencer depends on this value. I can't test it without that.

Reply:
What's the benefit of having it in a separate PR?

Reply:
Because developing the correct smart contract functionality is orthogonal to the DA node part.

Reply:
it's just adding one field, checking signature etc can be in separate PR, but this will just complicate things

Reply:
The DA part in the smart contract isn’t trivial either.

Reply:
yeah so DA part can be separate PR, here it's just adding location to the state, making separate PR for that would complicate it

Reply:
Why do you need it in the smart contract state though?

Reply:
Where should it be?

Reply:
I’m asking why you need it right now for testing. Smart contract has nothing to do with DA at all right now.

Reply:
The sequencer reads the location of the latest batch from the smart contract.

Reply:
just fake it for now?